target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -220,6 +220,10 @@ iodata,wn-ac733gr3)
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "$boardname:green:wlan5g" "phy0radio"
 	ucidef_set_led_wlan "wlan2g" "WLAN2G" "$boardname:green:wlan2g" "phy1radio"
 	;;
+iodata,wnpr2600g)
+	ucidef_set_led_wlan "wlan2g" "WLAN2G" "$boardname:green:wlan2g" "phy0radio"
+	ucidef_set_led_wlan "wlan5g" "WLAN5G" "$boardname:green:wlan5g" "phy1radio"
```

Comment:
These triggers can be set in dts using linux,default-trigger property.
BTW Is there a phy0tpt/phy1tpt trigger available? If there is, you could use tpt trigger which will also indicate active traffic.

Reply:
I checked *trigger* in each LEDs (/sys/class/leds/*/trigger), but there is no ```phy*tpt``` trigger.

>These triggers can be set in dts using linux,default-trigger property.

OK, I'll try it.